includes/class-comment.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Webmention;
+
+/**
+ * Undocumented class
+ */
+class Comment {
+	/**
+	 * Initialize the plugin, registering WordPress hooks
+	 *
+	 * @return void
+	 */
+	public static function init() {
+		self::register_comment_types();
+	}
+
+	/**
+	 * Register the comment types used by the Webmention plugin
+	 *
+	 * @return void
+	 */
+	public static function register_comment_types() {
```

Comment:
Should you register comment? The reason being the idea of having the comment type registry is to be able to execute per type theming, which WordPress itself doesn't have. Like the idea of switching back to comment instead of reply.

Reply:
Do we really want to change the default output of `comments`, `pingbacks` and `trackbacks`?

Reply:
I was thinking only if protocol is webmention

Reply:
the comment marker made issues lately because the core HTML changed a bit and so it broke themes. I would not recommend to overwrite the comment templates using the walker!